1ES-Provision.yml

```diff
@@ -0,0 +1,127 @@
+trigger:
```

Comment:
This file seem to do the same thing as `azure-pipeline-v3.yml`, except for some trigger and parameter differences, could it be merged as one?

Reply:
Actually it's very different from `azure-pipeline-v3.yml` as we should promise all repos be downloaded to each agent.

Reply:
The bottom line is to share the same list of repos between these two jobs.

Reply:
Seems difficult to share. Currently azure-pipeline-v3.yml contains no branch info but for warm-up we need branch or the clone will fail.

Reply:
I found that `parameters` for azure pipeline may be useful for this case. Testing...

Reply:
Done